webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/TechniqueEditForm.scala

```diff
@@ -218,7 +218,9 @@ class TechniqueEditForm(
             (
               "#techniqueName" #> t.name &
               "#techniqueDescription *" #> Script(
-                OnLoad(JsRaw(s"""generateMarkdown(${Str(t.description).toJsCmd}, "#techniqueDescription")"""))
+                OnLoad(
+                  JsRaw(s"""generateMarkdown(${Str(t.description).toJsCmd}, "#techniqueDescription")""")
+                ) // JsRaw ok, escaped
```

Comment:
I think `t.description` still needs to be escaped

Reply:
In the end, it seems OK, the  `Str(...).toJsCmd` does "Unicode escaping", which is another kind of escaping...